aspnetcore/data/ef-mvc/intro.md

```diff
@@ -383,7 +388,7 @@ Advance to the next tutorial to learn how to perform basic CRUD (create, read, u
 
 [!INCLUDE [RP better than MVC](~/includes/RP-EF/rp-over-mvc.md)]
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 2.2 MVC web applications using Entity Framework (EF) Core 2.2 and Visual Studio 2017 or 2019.
+The Contoso University sample web application demonstrates how to create ASP.NET Core 6.0 MVC web applications using Entity Framework (EF) Core 6.0.9 and Visual Studio 2022 or 2022.
```

Comment:
This is in the moniker section for earlier than 3.1, so it can't be updated to .NET 6 / VS2022.

```suggestion
The Contoso University sample web application demonstrates how to create ASP.NET Core 2.2 MVC web applications using Entity Framework (EF) Core 2.2 and Visual Studio 2019.
```